client/i18n/locales/english/intro.json

```
@@ -2433,7 +2433,7 @@
       "lecture-working-with-css-variables": {
         "title": "Working with CSS Variables",
         "intro": [
-          "In these lecture videos, you will learn how to define and use custom properties, AKA CSS variables. You will also learn about the <code>@property</code> rule and how it works."
+          "In these lecture videos, you will learn how to define and use custom properties, or CSS variables. You will also learn about the <code>@property</code> rule and how it works."
```

```suggestion
          "In these lecture videos, you will learn how to define and use custom properties (also known as CSS variables). You will also learn about the <code>@property</code> rule and how it works."
```
This makes it clear that CSS variables and custom properties are one and the same.